src/ui_manager.py

```diff
@@ -10,9 +10,9 @@
 import os
 import numpy as np
 from logger import Logger
-from utils.misc import wait, cut_roi
+from utils.misc import wait, cut_roi, color_filter, send_discord
 from config import Config
-from utils.misc import color_filter
+import threading
```

Comment:
why import threading here?

Reply:
That was for my custom discord hook to see if hes actually stashing :) just forgot to delete the import